src/WorkflowsRules/Services/Rules.php

```diff
@@ -51,19 +49,14 @@ public function validate(WorkflowsEntityInterfaces $entity, ...$args) : bool
         );
         if ($result) {
             $actions = $this->rule->getRulesActions();
+            $workflowLog = new Logs();
```

Comment:
@FredPeal this shouldnt be log but a workflow thread? because we want to keep a record withing this loop

Reply:
The thread start with the method start in the log service, this is called when the package validated the conditions for this rule. If the condition is succeeded the logs start with the rule and each action of this rule, is append to thread